src/routers/project.js

```diff
@@ -4,11 +4,11 @@ const router = express.Router();
 
 const project = require('../controllers/project')
 
-router.get('/project', project.getAllProjects)
-router.get('/project/:id', project.getProjectByID)
-router.get('/project/:projectid/comment/', project.getAllComments)
-router.get('/project/:projectid/comment/:commentid', project.getCommentByID)
-router.get('/project/:projectid/review/', project.getAllReviews)
-router.get('/project/:projectid/review/:remiewid', project.getReviewByID)
+router.get('/', project.getAllProjects)
+router.get('/:id', project.getProjectByID)
+router.get('/:id/comments/', project.getAllComments)
+router.get('/:id/comments/:commentid', project.getCommentByID)
+router.get('/:id/reviews/', project.getAllReviews)
+router.get('/:id/reviews/:remiewid', project.getReviewByID)
```

Comment:
type => :reviewID ***

Reply:
lowerCamelCase for all multi words variables commentId, reviewId, userId, etc

Reply:
![image](https://user-images.githubusercontent.com/14183741/151967141-bb010bc0-c6c2-4302-a317-46d9c3646506.png)

Reply:
OMG typoes never end